azcli_conformity.sh

```diff
@@ -0,0 +1,60 @@
+# Define Variables for the Configurations
+export APP_SECRET=$(uuidgen)
```

Comment:
@brianlamCC Is `uuidgen` an acceptable tool to use for generating a customers secret? This will be used to connect their Azure subscriptions to Conformity.